libraries/script-engine/src/AssetScriptingInterface.h

```diff
@@ -24,6 +26,7 @@
 #include <BaseScriptEngine.h>
 #include <QtNetwork/QNetworkDiskCache>
 
+/// Provides the <code><a href="https://apidocs.vircadia.dev/Assets.html">Assets</a></code> scripting API
```

Comment:
For items that have both Doxygen and JSDoc, I think that the Doxygen comments should be after JSDoc comments: the Doxygen comments will be closer to the code whereas the JSDoc comments will be more "distant" in that they interpret the code for JavaScript API users.